cmd/ipfs-cluster-service/main.go

```diff
@@ -190,9 +190,15 @@ func main() {
 			Usage: "enable full debug logging (very verbose)",
 		},
 		cli.StringFlag{
-			Name:  "loglevel, l",
-			Value: defaultLogLevel,
-			Usage: "set the loglevel for cluster components only [critical, error, warning, info, debug]",
+			Name:   "loglevel, l",
+			Value:  defaultLogLevel,
+			EnvVar: "LOG_LEVEL",
```

Comment:
CLUSTER_LOG_LEVEL

Reply:
`LOG_LEVEL` matches the flag name.
 One could get confused thinking `CLUSTER_LOG_LEVEL` affects only cluster component.

Reply:
`LOG_LEVEL` matches the flag name.
 One could get confused thinking `CLUSTER_LOG_LEVEL` affects only cluster component.